src/applications/check-in/api/versions/v2.js

```diff
@@ -6,15 +6,21 @@ import environment from '@department-of-veterans-affairs/platform-utilities/envi
 import { makeApiCallWithSentry } from '../utils';
 
 const v2 = {
-  getSession: async ({ token, checkInType }) => {
+  getSession: async ({
+    token,
+    checkInType = 'day-of',
+    setECheckinStartedCalled = false,
+  }) => {
     const url = '/check_in/v2/sessions/';
     let requestUrl = `${environment.API_URL}${url}${token}`;
     if (checkInType) {
       requestUrl = appendQuery(requestUrl, {
         checkInType,
       });
     }
-    const eventLabel = `${checkInType || 'day-of'}-get-current-session-dob`;
+    const eventLabel = `${checkInType || 'day-of'}-get-current-session-dob${
```

Comment:
Just checking, but this is something like -session-dob45 and not meant to be -session-dob-45 right?

Reply:
Whoops! I meant to remove that since it's not possible to pass that parameter into the call to the session endpoint.